-std=gnu++2a works fine with old Clang if libc++ is new enough

CMake Error at CMakeLists.txt:11 (message):
  RPCS3 requires at least clang-9.0.

--- CMakeLists.txt.orig	2020-02-28 18:59:56 UTC
+++ CMakeLists.txt
@@ -7,8 +7,8 @@ if(CMAKE_COMPILER_IS_GNUCXX)
 		message(FATAL_ERROR "RPCS3 requires at least gcc-9.")
 	endif()
 elseif(${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
-	if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0)
-		message(FATAL_ERROR "RPCS3 requires at least clang-9.0.")
+	if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 8.0)
+		message(FATAL_ERROR "RPCS3 requires at least clang-8.0.")
 	endif()
 endif()
 
